### DISCUSSION TITLE
re-build PR if target branch is changed

### DISCUSSION THREAD

**Discussion by carbohydrates** [community/community#26231] [Actions]
<p>Hello folks,</p>
<p>I want to have an ability to rebuild all open pull requests  if the target branch is changed(ex. one of the opened pull requests was merged)</p>
<p>So how can I manage that?</p>
<p>Thanks in advance</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>It’s not supported to re-build PR check runs if target branch is changed.</p>
<p>For pull request event, it monitors the last <a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request" rel="nofollow noopener">MERGE</a> commit on the github_ref branch, which won’t affect other branches which also created pull request, they are separated.</p>
